parsers/miniseq/sample_parser.py

```diff
@@ -287,11 +288,15 @@ def _parse_samples(sample_sheet_file):
 
         new_sample_dict = deepcopy(sample_dict)
         new_sample_name = new_sample_dict['sampleName']
+        new_sample_desc = ''
+        if 'Description' in new_sample_dict:
+            new_sample_desc = new_sample_dict['description']
```

Comment:
Throw a comment in here for why we're adding this here please.

Reply:
Added.